src/Carbon/Carbon.php

```diff
@@ -1576,6 +1576,16 @@ public function isLeapYear()
     }
 
     /**
+     * Determines if the instance is a long year
+     *
+     * @return bool
+     */
+    public function isLongYear()
+    {
+        return static::create($this->year, 12, 28, 0, 0, 0, $this->tz)->weekOfYear === 53;
```

Comment:
Nah, I don't think that's necessary - I've read it too
https://en.wikipedia.org/wiki/ISO_8601#Week_dates is already self-explanatory IMO